polymodels/fields.py

```diff
@@ -10,12 +10,16 @@
 )
 from django.utils.deconstruct import deconstructible
 from django.utils.functional import LazyObject, empty
-from django.utils.six import string_types
 from django.utils.translation import ugettext_lazy as _
 
 from .models import BasePolymorphicModel
 from .utils import get_content_type
 
+try:
+    from six import string_types
+except ImportError:
+    from django.utils.six import string_types
```

Comment:
This case is unreachable since the django-polymodels (in this branch version) is dependent on six
https://github.com/kmmbvnr/django-polymodels/blob/a16c21efac622d34302fbfaf53964826e444cfe8/setup.py#L17-L20

We can use ever direct import from six or discuss the possibility of Python2 (with some old django versions) support dropping.

Same in other files, I will not duplicate the comment

Reply:
I think adding/removing python versions is out of scope of Django 3.0 support